code/formality_genre_analysis.py

```diff
@@ -0,0 +1,50 @@
+import pandas as pd 
+import numpy as np 
+from matplotlib import pyplot as plt 
+import os 
+import seaborn as sns 
+from scipy import stats
+from collections import Counter
+
+# Specify the path to the folder
+folder_path = r"C:\Users\rasik\Documents\Independent Study\data\formality_genre_data"
+
+# List all files and directories in the folder
+files = os.listdir(folder_path)
```

Comment:
this can be dangerous because the list of things you are processing is not explicit. if some files are somehow missing or mistakenly added, it can break the workflow. If possible, be explicit about the list of files that you're processing.